### ISSUE TITLE
Monitor background color different from the rest of Weave Cloud

### ISSUE THREAD

**Issue by bia** [weaveworks/ui-components#167]
Elaborating from #93, now that we are measuring legibility in a quantitative way, [WCAG 2 AA Compliance](https://snook.ca/technical/colour_contrast/colour.html#fg=33FF33,bg=333333).

While most of the app's background color is `#fafafc`, Explore is `#f8f8f8` _lightgray_ and Monitor is `#ffffff` with `#f8f8f8` _lightgray_ overlayed blocks.
Both backgrounds should be changed to `#fafafc` for consistency.

If we do shift the Monitor background to be consistent with the rest of the app, we might consider shifting lightgray to some darker variant of `#fafafc` with the same hue. For example, `#ededf0` (`#fafafc` with 5% less black).

Related to #65 
cc @miklosp
